codespell_lib/data/dictionary.txt

```diff
@@ -5594,7 +5595,13 @@ atleast->at least
 atll->all
 atmoic->atomic
 atmoically->atomically
+atomate->automate
+atomated->automated
+atomates->automates
+atomatic->automatic
```

Comment:
It could certainly be `aromatic`, couldn't it?

Perhaps `atomic` too, although far-fetched.

Reply:
`atomic` and `aromatic` are useful.